server/clearbit.js

```diff
@@ -154,32 +160,32 @@ export default class Clearbit {
     const domain = getDomain(user);
 
     if (!this.client || !discover_enabled || _.isEmpty(discover_segments)) {
-      this.debug("Discover not enabled", { discover_segments });
+      this.hull.asUser(_.pick(user, ["id", "external_id", "email"])).logger.info("outgoing.user.skip", { reason: "Discover not enabled", discover_segments, action: "discover" });
```

Comment:
you can initialize the user scoped logger at the beginning of the method here instead of repeating the same `asUser` line each time ;)